### DISCUSSION TITLE
How to Customize Filament Dashboard?

### DISCUSSION THREAD

**Discussion by kinjal1298** [filamentphp/filament#13236] [Help] [answered]
### Package

Other

### Package Version

v3,2

### How can we help you?

I want to use another chart package in my Filament project. I want to display charts under the widgets on the dashboard. How can I render my Blade file there? Please help if anyone has an idea.
![ams_dashboard1](https://github.com/filamentphp/filament/assets/83871039/0c7e06da-bd81-4eac-8d25-c464f717cda8)

**Comment by griggsk** [comment]
I think you would create a custom widget: https://filamentphp.com/docs/3.x/widgets/installation
```<?php

namespace App\Filament\Widgets;

use Filament\Widgets\Widget;

class CustomWidget extends Widget
{


    protected static bool $isLazy = false;

    /**
     * @var view-string
     */
    protected static string $view = 'widgets.custom';
}
```
custom.blade.php
```<x-filament-widgets::widget> ... </x-filament-widgets::widget>```

**Reply by kinjal1298** [reply]
Thank you so much @griggsk ,
I tried adding custom Blade files this way as well. It works.

**Comment by leandrocfe** [comment, marked as answer]
This is the correct link if you are trying to add a custom widget:
https://filamentphp.com/docs/3.x/panels/dashboard#custom-widgets

What kind of charts do you want to add?
Have you ever seen ApexCharts Plugin?
https://github.com/leandrocfe/filament-apex-charts

**Reply by kinjal1298** [reply]
Thank you so much @leandrocfe for your response and for suggesting the plugin. I really appreciate your help and will look into using them to solve my issue.
